dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -160,6 +167,10 @@ public WorkflowResource(HttpClient client, SessionFactory sessionFactory, Permis
         zenodoUrl = configuration.getZenodoUrl();
         zenodoClientID = configuration.getZenodoClientID();
         zenodoClientSecret = configuration.getZenodoClientSecret();
+
+        hostName = configuration.getExternalConfig().getHostname();
+        scheme = configuration.getExternalConfig().getScheme();
+        port = configuration.getExternalConfig().getPort();
```

Comment:
As per previous comment, this should really be UI port 
(The generated URL should be to a UI URL as opposed to the webservice directly)

Reply:
done